.github/workflows/e2e-tests.yml

```diff
@@ -16,8 +16,18 @@ jobs:
         with:
           node-version: '12.x'
 
+      - name: Cacheando dependencias
```

Comment:
@LorhanSohaky, o cache implementado no job `cache_dependencies` é individual para cada execução ou é compartilhado entre as várias execuções da pipeline?

Se o cache é individual para cada execução, entendo que não faz sentido termos o cache, pois não trás ganho para o estado atual da pipeline.

Reply:
O que entendi é que se não tiver alteração no package lock ele deve utilizar o mesmo cache segundo essas restrições

https://docs.github.com/pt/actions/using-workflows/caching-dependencies-to-speed-up-workflows#restrictions-for-accessing-a-cache